src/rules/selector-no-type/__tests__/index.js

```diff
@@ -158,6 +158,24 @@ testRule(rule, {
   accept: [
     { code: "// Comment\n.c {}" },
   ],
+
+  reject: [{
+    code: ".foo { div {} }",
+    description: "nested descendant",
+    message: messages.rejected,
```

Comment:
Can we add the warning position to these rejected cases to ensure that it's correct?